ref/general/packaging-deploying.adoc

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: You can choose how to package and deploy your application with Open Liberty to meet your needs.
+:seo-title: Packaging and deploying applications
+:seo-description: You can choose how to package and deploy your application with Open Liberty to meet your needs.
+:page-layout: general-reference
+:page-type: general
+= Package and deploy applications with Open Liberty
+
+You can choose how to package and deploy your application with Open Liberty to meet your needs.
+
+== Package, containerize, and deploy using Maven and Docker
+
+One of the easiest ways to get your cloud-native application up and running is to use Maven to package it with the Open Liberty runtime and use Docker to containerize and deploy it.
+The Open Liberty Maven plug-in includes goals that allow you to manage your Open Liberty server and applications.
+Go to the link:https://github.com/OpenLiberty/ci.maven[documentation] for information about how to enable the Open Liberty Maven plug-in for your project.
+Use the latest Open Liberty Docker images for access to the most recent feature versions and link:/docs/ref/general/#security-vulnerabilities.html[security fixes].
+You can download Open Liberty Docker images from link:https://hub.docker.com/_/open-liberty[Docker Hub].
+
+Maven is a build automation tool that makes managing your builds and dependencies easier.
+You use a Project Object Model (POM) file to define information about your project, and Maven downloads and installs all dependencies for you.
+To learn more about using Maven with Open Liberty, check out this guide on link:/guides/maven-intro.html[Building a web application with Maven].
+
+When you're ready to package your application, run the `mvn package` command.
+This command compiles and builds the project.
+Then, you can use the `docker build` and `docker run` commands to build your image and run it as a container.
+To learn more about running your applications with the Open Liberty Docker images, see this guide on link:/guides/containerize.html[Containerizing microservices].
+
+== Package into a JAR file using Maven
+
+In some cases, you might want to package you application into a JAR file.
```

Comment:
what are these cases? why would you want to? I think there's some mention of it in the pack and deploy guide

Reply:
Also- change `you application` to `your application`